lib/V8/v8-globals.h

```diff
@@ -740,6 +740,20 @@ struct TRI_v8_global_t {
   /// @brief the current security context
   arangodb::JavaScriptSecurityContext _securityContext;
 
+  /// @brief true if the arango infrastructure is garbage collecting
+  bool _inForcedCollect;
+
+  /// @brief the ID that identifies this v8 context
+  size_t _id;
```

Comment:
Can the id ever change?
```suggestion
  size_t const _id;
```